python/doc/examples/functional_modeling/vectorial_functions/plot_functions_outputDim.py

```diff
@@ -0,0 +1,79 @@
+"""
+===========================================
+Increase the output dimension of a function
+===========================================
+"""
+# %%
+#
+# Import the required modules
+
+# %%
+import openturns as ot
+
+# %%
+# Description
+# ===========
+#
+# We want to build a function :math:`f : \mathbb{R}^d \mapsto \mathbb{R}^q` from *q* functions :math:`f_i: \mathbb{R}^d \mapsto \mathbb{R}`.
+#
+# We can do that:
+#
+#     - Case 1: by aggregation  of the functions :math:`f_i`,
+#     - Case 2: by creating a vectorial linear combination  of the functions :math:`f_i`.
+
+
+# %%
+# Case 1: Aggregation
+# ===================
+#
+# We have *q* functions :math:`f_i : \mathbb{R}^d \mapsto \mathbb{R}`.
+#
```

Comment:
![image](https://user-images.githubusercontent.com/31351465/230657690-dab4174a-10c1-4c99-a85e-4781805fb51a.png)

It is not necessary to create a new line here.
```suggestion
```

Reply:
ok